frontend/src/static/js/pages/styles/PlaylistPage.scss

```
@@ -474,6 +474,13 @@
 	.playlist-actions{
 
 		position:relative;
+		margin-top:12px;
+		// Flex row: SHARE aligned left, three-dot menu pushed to the right
```

Stylelint (17.6.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)